webui/src/docs/rest-api.jsx

```diff
@@ -1,244 +1,540 @@
+                "next": "https://vm0045.kaj.pouta.csc.fi/api/records/?sort=bestmatch&q=community%3Ae9b9792e-79fb-4b07-b6b4-b9c2bd06d095&size=10&page=2",
+                "self": "https://vm0045.kaj.pouta.csc.fi/api/records/?sort=bestmatch&q=community%3Ae9b9792e-79fb-4b07-b6b4-b9c2bd06d095&size=10&page=1"
+              }
+            }}
+            </Returns>
+        </Example>
+
+        <h3>Search records</h3>
+        <p>Search all the published records for a query string.</p>
+        <ul>
+            <li><p>URL path: /api/records/?q=$QUERY_STRING</p></li>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>Required parameters: access_token</p></li>
+            <li><p>Returns: the list of matching records (in JSON format) or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/records/?q=$QUERY_STRING?access_token=$ACCESS_TOKEN
+        </Example>
+
+        <h3>Search drafts</h3>
+        <p>Search for all drafts (unpublished records) that are accessible by the requestor. Usually this means own records only.</p>
+        <ul>
+            <li><p>URL path: /api/records/?drafts=1</p></li>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>Required parameters: access_token, drafts</p></li>
+            <li><p>Returns: the list of matching drafts (in JSON format) or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/records/?drafts=1?access_token=$ACCESS_TOKEN
+        </Example>
+
+        <h3>Get a specific record</h3>
+        <p>List the metadata of the record specified by RECORD_ID</p>
+        <ul>
+            <li><p>URL path: /api/record/RECORD_ID</p></li>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>Required parameters: access_token</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/records/47077e3c4b9f4852a40709e338ad4620?access_token=$ACCESS_TOKEN
+        </Example>
+
+
+        <h3 id="create-draft">Create a draft record</h3>
+        <p>Create a new record, in the draft state.</p>
+        <ul>
+            <li><p>URL path: /api/records</p></li>
+            <li><p>HTTP method: POST</p></li>
+            <li><p>Required URL parameter: access_token</p></li>
+            <li><p>Required data payload: json object with basic information about the object </p></li>
+            <li><p>Returns: the new draft record contents and location. Please note that
+                the returned json object contains also the URL of the file bucket used for the record.
+                Also note that the URL of the draft record, needed for setting record metadata,
+                will end in '/drafts/'</p></li>
+        </ul>
+        <Example>
+            {'curl -i -H "Content-Type:application/json" -d \'{"title":"TestRest", "community":"e9b9792e-79fb-4b07-b6b4-b9c2bd06d095", "open_access":true}\' -X POST https://$HOSTNAME/api/records/?access_token=$ACCESS_TOKEN'}
```

Comment:
No, this is valid, just part of the jsx special syntax. There is a corresponding bracket at the end.